core/src/main/scala/org/apache/spark/sql/streaming/SnappySinkCallback.scala

```diff
@@ -107,7 +108,18 @@ case class SnappyStoreSink(snappySession: SnappySession,
       }
     }
 
-    sinkCallback.process(snappySession, parameters, batchId, convert(data), posDup)
+    val hashAggregateSize = Property.HashAggregateSize.get(snappySession.sessionState.conf)
+    val hashAggregateSizeSetToDefault = hashAggregateSize.equals("0")
```

Comment:
Use Property.HashAggregateSize.defaultValue.get instead of "0". Also the name of variable seems a bit other way. Either like "setHashAggregateSizeToDefault" or "hashAggregateSizeChanged"